www/doc/1.5/faq.html

```diff
@@ -112,6 +112,15 @@ <h3 id="egg-herbie">Falling back on regraph because egg-herbie package not insta
     Install <code>egg-herbie</code> with <code>raco</code>.
   </p>
 
+  <h3 id="unsound-rules">Unsound rule application detected</h3>
+  
+  <p>
+    Herbie uses a set of algebraic rewrite rules in order to simplify expressions.
+    Herbie can sometimes detect when these rewrite rules produce a contradiction.
+    Specifically, if it proves two constants or variables are equal, the rewrite rules
+    have been applied in an unsound way.
```

Comment:
This is an ok description of why this happens, but the user is interested in "what should I do".

Reply:
Best to say something like, "Herbie will attempt to automatically compensate for this, and in most cases nothing needs to be done. But it can be worth double-checking that derivation to make sure Herbie's output is what you want."